docs/src/layouts/links.js

```diff
@@ -461,7 +461,7 @@ const ComponentsLinks = [
         id: 'FLARE',
         route: routes.FLARE,
         children: 'Flare',
-        tags: ['web', 'ios'],
+        tags: ['web', 'ios', 'native'],
```

Comment:
Android seems to be missing here, can you check whilst you are changing this?

Reply:
Sure, good catch!